packages/scss/src/components/button/component.scss

```diff
@@ -19,14 +23,10 @@
 	color: var(--palettes-text, var(--palettes-primary-text));
 	cursor: pointer;
 
-	.lucca-icon {
+	.button-icon, .lucca-icon { // .lucca-icon legacy
```

Comment:
where does this class come from ?

Reply:
It used to be mandatory on `button mod-icon` but we slowly started to target `.lucca-icon`, which is a bad practice considering [our guidelines](https://prisme.lucca.io/94310e217/p/929c63-guidelines-intgration/t/30c6d8). So we reintroduced this class and considered `.lucca-icon` as deprecated.

Reply:
considering to [this guidelines](https://prisme.lucca.io/94310e217/p/929c63-guidelines-intgration/t/57f49d) it seems to be normal to have `.lucca-icon` and not a new class as it's en encapsulation component, isn't it ?